src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -3980,6 +4000,83 @@ protected override RedisValue[] Parse(in RawResult result, out int count)
             }
         }
 
+        private static SortedSetRangeAndStoreCommandMessage CreateSortedSetRangeStoreMessage(
+            int db,
+            CommandFlags flags,
+            RedisKey destinationKey,
+            RedisKey sourceKey,
+            RedisValue start,
+            RedisValue stop,
```

Comment:
If we're converting the `(`/`[` prefixes internally, can these be `long` instead through all overloads or am I missing a case/use?